tests/functionnal/Glpi/Inventory/Inventory.php

```diff
@@ -1436,13 +1436,13 @@ public function testUpdateComputer()
             'LIMIT' => $nblogsnow,
             'OFFSET' => $this->nblogs,
         ]);
-        $this->integer(count($logs))->isIdenticalTo(72);
+        $this->integer(count($logs))->isIdenticalTo(74);
 
         $expected_types_count = [
             \Log::HISTORY_CREATE_ITEM => 64,
             \Log::HISTORY_ADD_SUBITEM => count($expecteds_fs),
             0 => 1, // Change Monitor contact (is_contact_autoupdate)
-            \Log::HISTORY_ADD_RELATION => 4 //OS and Monitor on both sides
+            \Log::HISTORY_ADD_RELATION => 6 // 2 IPNetwork/IPAdresss + OS and Monitor on both sides
```

Comment:
Now, 2 IPNetwork/IPAddress items are created.

First one correspond to 
```json
            {
                "description": "enp0s25",
                "driver": "e1000e",
                "ipaddress": "172.28.220.96",
                "ipgateway": "172.28.220.1",
                "ipmask": "255.255.255.0",
                "ipsubnet": "172.28.220.0",
                "mac": "b8:6b:23:e0:5c:81",
                "pciid": "8086:1559:1179:0001",
                "pcislot": "0000:00:19.0",
                "speed": "100",
                "status": "up",
                "type": "ethernet",
                "virtualdev": false
            },
```

Second one corresponds to
```json
            {
                "description": "wlp2s0",
                "driver": "iwlwifi",
                "ipaddress": "172.28.220.181",
                "ipgateway": "172.28.220.1",
                "ipmask": "255.255.255.0",
                "ipsubnet": "172.28.220.0",
                "mac": "30:3a:64:ee:65:88",
                "pciid": "8086:08B3:8086:8170",
                "pcislot": "0000:02:00.0",
                "status": "up",
                "type": "wifi",
                "virtualdev": false,
                "wifi_bssid": "00:1D:7E:43:96:59",
                "wifi_mode": "Managed",
                "wifi_ssid": "DaWifi",
                "wifi_version": "802.11"
            },
```

See https://raw.githubusercontent.com/glpi-project/inventory_format/master/examples/computer_3.json